wordpress-mobile/WordPress-iOS
# Remove obsolete bottom sheet demo

This PR removes the bottom sheet demo described in `p9wMUP-bZ-p2`, as it is no longer needed.

To test:

- Checkout the branch and confirm that existing tests pass.
- Review the changed code and make sure everything looks okay. It might be helpful to juxtapose these changes with the ones where the demo [was introduced](https://github.com/wordpress-mobile/WordPress-iOS/pull/10382).
- For a `localDeveloper` build, navigate to _My Sites_ and tap **+**. Verify that the existing options still work as before.

Update release notes:
